pr-data.csv

```diff
@@ -4684,8 +4684,8 @@ https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd14
 https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd1414ee4d4,spring-boot-project/spring-boot-actuator,org.springframework.boot.actuate.metrics.web.servlet.WebMvcMetricsIntegrationTests.rethrownExceptionIsRecordedInMetricTag,NIO,,,https://github.com/TestingResearchIllinois/idoft/issues/411
 https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd1414ee4d4,spring-boot-project/spring-boot-autoconfigure,org.springframework.boot.autoconfigure.gson.GsonAutoConfigurationTests.customGsonBuilder,ID,InspiredAFix,https://github.com/spring-projects/spring-boot/pull/30514,
 https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd1414ee4d4,spring-boot-project/spring-boot-autoconfigure,org.springframework.boot.autoconfigure.websocket.servlet.WebSocketMessagingAutoConfigurationTests.basicMessagingWithJsonResponse,ID,InspiredAFix,https://github.com/spring-projects/spring-boot/pull/30514,
-https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd1414ee4d4,spring-boot-project/spring-boot,org.springframework.boot.context.properties.bind.CollectionBinderTests.bindToSetShouldNotAllowDuplicateValues,ID,,,
-https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd1414ee4d4,spring-boot-project/spring-boot,org.springframework.boot.diagnostics.analyzer.BeanCurrentlyInCreationFailureAnalyzerTests.cycleWithAutowiredFields,ID,,,
+https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd1414ee4d4,spring-boot-project/spring-boot,org.springframework.boot.context.properties.bind.CollectionBinderTests.bindToSetShouldNotAllowDuplicateValues,ID,MovedToGradle,,
```

Comment:
It'd be nice (but not necessary) to know when the project (or even more precisely the module/test) moved from Maven to Gradle.

Reply:
I read through Wiki and all release notes, and I believe they migrated to Gradle in about 2021. There are about 46000 commits, and I cannot find the exact commit they migrated from Maven to Gradle.

Reply:
Please see my email.